src/header.tsx

```diff
@@ -26,25 +26,45 @@ type RichText = string | ({text: string} & OverridableTextProps);
 type HeaderProps = {
     pretitle?: RichText;
     title?: string;
+    description?: string;
+    dataAttributes?: DataAttributes;
+    /**
+     * @deprecated This field is deprecated, please use the extra slot in the HeaderLayout component instead.
```

Comment:
I'd remove deprecated props from the story too

Reply:
also from playroom snippets pls!